subsys/spm/spm.c

```diff
@@ -168,11 +181,19 @@ static void spm_config_flash(void)
 		PRINT("%c", flash_perm[i] & FLASH_LOCK  ? 'l' : '-');
 		PRINT("\n");
 	}
-	PRINT("\n");
 
 #if defined(CONFIG_ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS)
+	int err;
+
 	spm_config_nsc_flash();
+	err = spm_secure_services_init();
```

Comment:
#ifdef CONFIG_SECURE_SERVICES ?

Then we can save space by not including the below string in non-secure service FW's.

Reply:
Done